docs/src/modules/components/AppSettingsDrawer.js

```diff
@@ -37,6 +37,9 @@ const styles = (theme) => ({
     '&$toggleButtonSelected': {
       color: `${theme.palette.primary.main}`,
       backgroundColor: `${alpha(theme.palette.primary.main, theme.palette.action.selectedOpacity)}`,
+      '&:hover': {
+        backgroundColor: `${alpha(theme.palette.primary.main, 0.12)}`,
```

Comment:
See https://github.com/mui-org/material-ui/blob/6f56be2a07185be06984476aed9f518cbbd6b87d/packages/material-ui-lab/src/TreeItem/TreeItem.js#L50-L58 for a good approach, I think that we miss the hover none and the design tokens for the alpha value.

Reply:
> theme.palette.action.selectedOpacity + theme.palette.action.hoverOpacity

Ah, I didn't think of adding them. 👍 

By comparison, an active list item _doesn't_ change shade when hovered.

Perhaps there should be a distinction between a toggle button that can be deselected (changes shade when hovered), and one that can't (doesn't change shade when hovered)?

Reply:
> By comparison, an active list item doesn't change shade when hovered.

It seems to do in my case: https://next.material-ui.com/components/lists/#selected-listitem. We marked it as handled in #10870. 

> Perhaps there should be a distinction between a toggle button that can be deselected (changes shade when hovered), and one that can't (doesn't change shade when hovered)?

Why not. This new design makes me think about https://ant.design/components/radio/#Radio/Radio.Button